### ISSUE TITLE
stereo-seq run using long time and  huge memory

### ISSUE THREAD

**Issue by xingyongma** [raphael-group/paste2#8]
When I run bin 50 of stereo-seq, there are 110,000 bins. paste2 requires hundreds of gigabytes of memory and runs for dozens of hours. Is there any way for me to reduce resource usage and increase speed? Can I subset a portion of the data to do it?

**Comment by x-h-liu** [comment]
Yes! If you random downsample the stereo-seq cells, PASTE2 should work on the downsampled slices and the computed transformation still applies to the original slices.
